shell.nix

```diff
@@ -93,6 +93,7 @@ let
     bzip2
     cacert
     editorconfig-core-c
+    dateutils
```

Comment:
This little tool is needed to easily calcluate date differences and get messages like:
"this file is X days old"